yarn-project/end-to-end/src/e2e_public_token_contract.test.ts

```diff
@@ -95,8 +91,8 @@ describe('e2e_public_token_contract', () => {
     expect(receipts.map(r => r.status)).toEqual(times(3, () => TxStatus.MINED));
     expect(receipts.map(r => r.blockNumber)).toEqual(times(3, () => receipts[0].blockNumber));
 
-    const balance = await cc.l2.loadPublic(contract.address, cc.l2.computeSlotInMap(balanceSlot, recipient.toField()));
-    expect(balance.value).toBe(mintAmount * 3n);
+    const balance = (await contract.methods.publicBalanceOf(recipient.toField()).view({ from: recipient }))[0];
```

Comment:
does this need to return an array?

Reply:
edit: just seen convo above

Reply:
No. That is a currently just how return values are provided. There are some todo's in the rpc to return abi-types etc instead. Will be adding issues for it.